src/main/java/joe/Main/Storage.java

```diff
@@ -0,0 +1,43 @@
+package joe.Main;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+/**
+ * Represents the storage of the tasks.
+ */
+public class Storage {
+
+    private static final String line =
```

Comment:
If this is a constant should it be all capitalised?
```suggestion
    private static final String LINE =
```

Reply:
Yeap you are right, they should be in screaming snake_case. Thanks for pointing it out 😄